### ISSUE TITLE
"The file must be included in at least one of the projects provided" even after using tsconfig.eslint.json trick

### ISSUE THREAD

**Issue by SwiftWinds** [iamturns/eslint-config-airbnb-typescript#199]
Here's my .eslintrc.js:

```js
module.exports = {
  parser: '@typescript-eslint/parser', // add the TypeScript parser
  parserOptions: { project: './tsconfig.eslint.json' },
  plugins: [
    'svelte3',
    '@typescript-eslint', // add the TypeScript plugin
  ],
  extends: 'airbnb-typescript/base',
  overrides: [ // this stays the same
    {
      files: ['*.svelte'],
      processor: 'svelte3/svelte3',
    },
  ],
  rules: {
    // ...
  },
  settings: {
    'svelte3/typescript': require('typescript'), // pass the TypeScript package to the Svelte plugin
    // ...
  },
};
```

my tsconfig.eslint.json:
```json
{
  "extends": "./tsconfig.json",
  "include": ["**/*.{js,json,ts,svelte}"]
}
```

my tsconfig.json:
```json
{
  "extends": "@tsconfig/svelte/tsconfig.json",

  "include": ["src/**/*"],
  "exclude": ["node_modules/*", "__sapper__/*", "public/*"]
}
```

However, when I open any file, such as `src/main.ts`, ESLint complains that "The file must be included in at least one of the projects provided." Could you please offer any suggestion as to why this is happening? Any help would be greatly appreciated! :)

**Comment by abhisheksett** [comment]
Facing the same issue with React project

**Comment by Avivhdr** [comment]
Same here
